docs/source/getting_started/platform.rst

```diff
@@ -55,39 +55,25 @@ Learn more about the different concepts in our `Glossary <https://app.lightly.ai
 Create a Dataset
 ----------------
 
+There are several different ways to create a dataset on the lightly platform.
 
-To create a new dataset of images with embeddings use the lightly :ref:`lightly-command-line-tool`:
+The baseline way is to upload your local dataset including all images or
+videos to the Lightly platform.
 
-.. code-block:: bash
-
-    lightly-magic trainer.max_epochs=0 token='YOUR_API_TOKEN' new_dataset_name='my-dataset' input_dir='/path/to/my/dataset'
-
-
-Images and embeddings can also be uploaded from a Python script. For this, you need to 
-have a numpy array of image embeddings, the filenames of the images, and categorical pseudo-labels.
-You can use the `save_embeddings` function to store them in a lightly-compatible CSV format and
-upload them from your Python code:
+If you don't have your data locally, but rather stored at a cloud provider like
+in an AWS S3 bucket, GCloud bucket or at Azure, you can create a dataset directly
```

Comment:
better: "you can create a dataset directly referencing the images in your bucket."